nmdc_automation/workflow_automation/sched.py

```diff
@@ -347,4 +346,5 @@ def main():  # pragma: no cover
 
 if __name__ == "__main__":  # pragma: no cover
     logging.basicConfig(level=logging.INFO)
-    main()
+    # site_conf and wf_file are passed in as arguments
+    main("site_configuration.toml", "workflows.yaml")
```

Comment:
How will metatranscriptomes be handled? That has a different yaml name.

Reply:
Fix main() argument handling